### PR TITLE
doc: minor changes

### PR DESCRIPTION
**Pull request by Isammoc**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -341,8 +357,13 @@ dependencyManagement {
...
 ----
-This options ensures that `4.0.51.Final` will be used only for `gatling` configurations, leaving other dependencies unchanged.
+
+These options ensure that `4.0.51.Final` will be used only for `gatling` configurations, leaving other dependencies unchanged.
 
 == Release a new version
 
 . Push a tag `vX.Y.Z`, where `X.Y.Z` is the version to publish.
+
+== Acknowledgements
```
Comment (eshepelyuk) [inline README.adoc:367]: You must mention @lkishalmi as a maintainer and current plugin repository owner.

**Review by eshepelyuk — CHANGES_REQUESTED** [review]
Could you this PR with only text changes, without any formating.
Since it's almost impossible to analyze the changes.

All formatting should be done in other PR or just directly commiting to `master`.

File: README.adoc [hunk 2]
```diff
@@ -51,44 +50,44 @@ The plugin provides bootstrap script that creates sample project with:
 * proper source file layout
 * sample https://gatling.io/docs/current/general/simulation_structure/[Simulation class], demonstrating sufficient `Gatling` functionality
 
-For this quickstart - `git` and `curl` must be instlled and available in `$PATH`.
+For this quickstart: `git` must be installed and available in `$PATH`, and a JDK must be installed and `$JAVA_HOME` configured.
```
Comment (eshepelyuk) [inline README.adoc:53]: I have some ideas about sample project repo. Let me check first and may be i would provide a PR to it. 

@Isammoc May you add me as a collaborator to that project ?

Comment (eshepelyuk) [reply README.adoc:53]: Also, issues and wiki should be disabled for demo repo.

**Comment by Isammoc** [comment]
@eshepelyuk 
I don't split formatting in another PR, but I separated commits. Should be easier to analyze, now.

File: README.adoc [hunk 3]
```diff
@@ -164,8 +169,8 @@ gatling {
 ----
 gatling {
   simulations = {
-    include "**/package1/*Simu.scala"    <1>
-    include "**/package2/*Simulation.scala"  <2>
+    include "**/package1/*Simu.scala"    //<1>
```
[the remark this answers is not in the document]
Comment (Isammoc) [reply README.adoc]: Yep, I confirm! (and it isn't a big change here ^^)

[the remark this answers is not in the document]
Comment (eshepelyuk) [reply README.adoc]: In fact, this is a better way for callouts, according to docs.
https://asciidoctor.org/docs/user-manual/#copy-and-paste-friendly-callouts

**Comment by eshepelyuk** [comment]
@Isammoc just repeating it here, seems you may missed it.

> Is it intentionnaly that this PR doesn't mention issue gatling/gatling#3940, that will be eventually closed by this PR ?

**Review by eshepelyuk — APPROVED** [review]

**Review by eshepelyuk — CHANGES_REQUESTED** [review]
@Isammoc let me explain why logn paragraphs and tables were created as multiple shorter lines instead of one big long line of text ( as you've formatted it).

This is done to improve and simplify editing of README from:

* github editor
* console editor ( like vim)

Where the visible width of the lines is usually around 80 characters. 

I'd be really appreciated if you would keep the paragraph and tables formatting as is.

File: README.adoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (Isammoc) [reply README.adoc]: I should do it for XML as well.

[the remark this answers is not in the document]
Comment (eshepelyuk) [reply README.adoc]: Yes, great idea. Thanks.

**Comment by Isammoc** [comment]
@eshepelyuk 
> Where the visible width of the lines is usually around 80 characters.

When editing it, I didn't see that rule was previously followed.
But I can enforce it (I personally like it).

My point is to make the whole document consistent.

> Is it intentionnaly that this PR doesn't mention issue gatling/gatling#3940, that will be eventually closed by this PR ?

I just added a Reference in the last commit message

**Comment by eshepelyuk** [comment]
> When editing it, I didn't see that rule was previously followed.
> But I can enforce it (I personally like it).
> 
> My point is to make the whole document consistent.

Yes, you're right. Was not explicitely set since I was alone working on the project :)
Let's add this to `.editorconfig` in this PR ?

> > Is it intentionnaly that this PR doesn't mention issue [gatling/gatling#3940](https://github.com/gatling/gatling/issues/3940), that will be eventually closed by this PR ?
> 
> I just added a Reference in the last commit message

Thank you !

**Comment by slandelle** [comment]
The documentation of the plugin must live in the [Gatling's central documentation](https://github.com/gatling/gatling/tree/master/src/sphinx/extensions), not in the git repo's README.

Here we can have a quickstart and a link to the official documentation.

**Comment by eshepelyuk** [comment]
> The documentation of the plugin must live in the [Gatling's central documentation](https://github.com/gatling/gatling/tree/master/src/sphinx/extensions), not in the git repo's README.
> 
> Here we can have a quickstart and a link to the official documentation.

ok, it's fine with me.

**Review by eshepelyuk — APPROVED** [review]
LGTM

**Pull request merged**